backend/src/test/java/ch/puzzle/okr/util/quarter/check/QuarterRangeCheckerTest.java

```diff
@@ -40,10 +40,10 @@ void nowIsInQuarterShouldReturnTrueIfNowMatchesQuarterStartDate() {
         LocalDate startDate = LocalDate.of(2024, 7, 1);
         Quarter quarter = quarter(startDate, null);
 
-        LocalDate nowMatchesQuarterStartDate = LocalDate.of(2024, 7, 1);
+        LocalDate nowMatchingQuarterStartDate = LocalDate.of(2024, 7, 1);
```

Comment:
Why were all these attributes renamed? Is there something wrong with the old name?

Reply:
Matches sounds like a boolean to me, but ig thats subjective